### DISCUSSION TITLE
[Question]: how to get dropdown item list for Search component?

### DISCUSSION THREAD

**Discussion by brian316** [carbon-design-system/carbon#11028] [Help] [answered]
### Question for Carbon

I want to use the search component but I want it to show a list of available items under it as the user types.

Just like it is implemented here in the search bar https://www.carbondesignsystem.com/

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)

**Comment by tay1orjones** [comment, marked as answer]
Hey @brian316 👋  There's no component available for this off the shelf. The search used on the website is a custom component. It uses `Search`, and augments it with a div of results using `role='menu'` containing an unordered list of results.
